framework/encode/parameter_encoder.h

```diff
@@ -78,7 +78,7 @@ class ParameterEncoder
     void EncodeFunctionPtr(T value)                                                                                   { EncodeValue(reinterpret_cast<format::AddressEncodeType>(value)); }
 
     template<typename Wrapper>
-    void EncodeHandleValue(typename Wrapper::HandleType value)                                                        { EncodeHandleIdValue(GetWrappedId<Wrapper>(value)); }
+    void EncodeHandleValue(typename Wrapper::HandleType value)                                                        { EncodeHandleIdValue(GetVulkanWrappedId<Wrapper>(value)); }
```

Comment:
Add `Vulkan` to the parameter encoder name as well? e.g., `EncodeHandleValue` -> `EncodeVulkanHandleValue`